libqtile/core/manager.py

```diff
@@ -634,6 +646,17 @@ def unmanage(self, wid: int) -> None:
                     c.group.remove(c)
             del self.windows_map[wid]
             self.core.update_client_list(self.windows_map)
+            self.core.update_client_stack()
+
+    def check_stacking(self, win):
+        """Triggers restacking if a fullscreen window loses focus."""
```

Comment:
Is this just used for fullscreen windows? It seems to be used more. Could you elaborate on the purpose of this function?

Reply:
Fullscreen windows have top priority for stacking *when focused*. So, if a full screen window loses focus, we need to reposition it in the stack.